IS0402Test.py

```diff
@@ -131,6 +132,24 @@ def test_03(self, test):
         self.post_super_resources_and_resource(test, "node", "test_03")
         return test.PASS()
 
+    def test_03_1(self, test):
+        """Registration API responds with correct Location header"""
+
+        self.check_api_v1_x(test)
+
+        data = self.copy_resource("node")
+        data["id"] = str(uuid.uuid4())
+        data["description"] = "test_03_1"
+
+        # most tests on the Location header are actually done in every call to post_resource
+        location_header = self.post_resource(test, "node", data, codes=[201])
+
+        # also check an 'https' URL in the Location header has a hostname not an IP address
+        if location_header.startswith("https://") and urlparse(location_header).hostname[-1].isdigit():
```

Comment:
Looks like `location_header` may be returned as `None` so might be best to check for that first? I assume it may throw an exception otherwise.

Reply:
I don't believe execution can reach the `return Node` when passing `codes=[200]` because it'll have already have raised an `NMOSTestException` for other status codes or if Location is not present.

Probably still worth adding `location is not None and` anyway.